Bamboo/frontend/bamboo-frontend/app/guards/auth_guard.tsx

```diff
@@ -96,42 +96,51 @@ export default function AuthGuard(
   useEffect(() => {
 
     // TODO PDB-90
-    console.log(jwtString)
-    // try{
-    //
-    //
-    //   // User has logged out
-    //   if(!jwtString){
-    //     UrlScopeRestriction(location.pathname, ['/login', '/register'],
-    //       ()=>{nav('/login')})
-    //   }
-    //
-    //   // User has JWT
-    //   if(jwtString && jwtString!==''){
-    //     const jwtPayload:JwtPayload = jwt.deserialise(jwtString)
-    //
-    //     switch (jwtPayload.userType){
-    //       case UserAccountTypes.Student:
-    //         nav('/student')
-    //         break
-    //       case UserAccountTypes.Tutor:
-    //         nav('/tutor')
-    //         break
-    //       case UserAccountTypes.Uninitialised:
-    //         nav('/welcome')
-    //         break
-    //       default:
-    //         console.log(jwtPayload)
-    //         nav('/login')
-    //         break
-    //     }
-    //
-    //   }
-    // } catch (e) {
-    //   console.error(e)
-    // }
-
-  }, [jwtString]);
+    // console.log(jwtString)
+    try{
+    
+    
+      // User has logged out
+      if(!jwtString){
+        UrlScopeRestriction(location.pathname, ['/login', '/register'],
+          ()=>{nav('/login')})
+      }
+
+      // Decode the token to access the exp claim
+      const decodedToken = jwt.decode(jwtString) as JwtPayload
+
+      if (decodedToken.exp && Date.now() >= decodedToken.exp * 1000) {
+        //Token is expired and clear it from storage
+        setJwtString('') // Clear the Jwt token
```

Comment:
Will need to clear httpOnly cookie too

Reply:
Added in routes to clear httpOnly cookie. Please help me review the recent changes